### PR TITLE
MSI: update path with proper value

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

MSI: update path with proper value
    - Add verification that path is updated

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: tools/appveyor.psm1 [hunk 1]
```diff
@@ -486,6 +486,16 @@ function Invoke-AppveyorFinish
             $preReleaseVersion = "$previewPrefix-$previewLabel.$env:APPVEYOR_BUILD_NUMBER"
         }
 
+        # Save any powerShell paths, so we can exclude them
```
Comment (SteveL-MSFT) [inline tools/appveyor.psm1]: PowerShell

File: tools/appveyor.psm1 [hunk 2]
```diff
@@ -499,6 +509,15 @@ function Invoke-AppveyorFinish
         }
         Write-Log "MSI smoke test was successful"
 
+        # Verify path was updated by MSI
+        $psPath = ([System.Environment]::GetEnvironmentVariable('PATH','MACHINE')) -split ';' |
```
Comment (SteveL-MSFT) [inline tools/appveyor.psm1]: Wouldn't the env var only be updated for a new process?

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: tools/appveyor.psm1 (hunk 2 above)
Comment (TravisEz13) [reply tools/appveyor.psm1]: that is why I'm reading it from the machine

File: tools/appveyor.psm1 [hunk 3]
```diff
@@ -499,6 +509,15 @@ function Invoke-AppveyorFinish
...
+            Where-Object {$_ -like '*files\powershell*' -and $_ -notin $beforePath}
```
Comment (iSazonov) [inline tools/appveyor.psm1]: What if we'll always install in the same directory as we discussed in RFC?

File: tools/appveyor.psm1 (hunk 2 above)
Comment (SteveL-MSFT) [reply tools/appveyor.psm1]: got it.  thanks

File: tools/appveyor.psm1 (hunk 3 above)
Comment (TravisEz13) [reply tools/appveyor.psm1]: this pattern still matches

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**